<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Rewind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel" native="true">
       <layout class="QVBoxLayout" name="leftPanelLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="gamesLabel">
          <property name="text">
           <string>Detected Games</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="gamesTreeWidget">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Game</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel" native="true">
       <layout class="QVBoxLayout" name="rightPanelLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="backupsLabel">
          <property name="text">
           <string>Backups</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="selectedGameLabel">
          <property name="text">
           <string>No game selected</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { color: palette(window-text); opacity: 0.6; }</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="backupsListWidget">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreateBackup"/>
   <addaction name="actionRestoreBackup"/>
   <addaction name="actionDeleteBackup"/>
   <addaction name="separator"/>
   <addaction name="actionAddGame"/>
   <addaction name="actionScanGame"/>
   <addaction name="actionManageConfigs"/>
   <addaction name="actionHiddenGames"/>
   <addaction name="separator"/>
   <addaction name="actionBackUpAll"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionCreateBackup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Create Backup</string>
   </property>
   <property name="toolTip">
    <string>Create a new backup of the selected game</string>
   </property>
  </action>
  <action name="actionRestoreBackup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-revert"/>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore the selected backup</string>
   </property>
  </action>
  <action name="actionDeleteBackup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete the selected backup</string>
   </property>
  </action>
  <action name="actionAddGame">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Add Game</string>
   </property>
   <property name="toolTip">
    <string>Add a custom game manually</string>
   </property>
  </action>
  <action name="actionScanGame">
   <property name="icon">
    <iconset theme="edit-find"/>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
   <property name="toolTip">
    <string>Scan for game save directories</string>
   </property>
  </action>
  <action name="actionManageConfigs">
   <property name="icon">
    <iconset theme="configure"/>
   </property>
   <property name="text">
    <string>Manage Configs</string>
   </property>
   <property name="toolTip">
    <string>Open the game configuration editor</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh the game list</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="settings-configure"/>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>View application settings</string>
   </property>
  </action>
  <action name="actionHiddenGames">
   <property name="icon">
    <iconset theme="view-hidden"/>
   </property>
   <property name="text">
    <string>Hidden Games</string>
   </property>
   <property name="toolTip">
    <string>Manage hidden games</string>
   </property>
  </action>
  <action name="actionBackUpAll">
   <property name="icon">
    <iconset theme="document-save-all"/>
   </property>
   <property name="text">
    <string>Back Up All</string>
   </property>
   <property name="toolTip">
    <string>Back up multiple games at once</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Game Rewind</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
